tools/linter.py

```diff
@@ -40,7 +40,7 @@ def run_pycodestyle(self, diff):
         """
             Original Author: Josh Wilson (@person142)
             Source:
-              https://github.com/scipy/scipy/blob/master/tools/lint_diff.py
+              https://github.com/scipy/scipy/blob/main/tools/lint_diff.py
```

Comment:
this link will change very soon when SciPy renames its master branch to main, so just fixing up preemptively